### PR TITLE
removed column status in Dispensation History, fine tunning in prescr…

### PR DESCRIPTION
**Pull request by Guimino**
Removed column status in Dispensation History. 
Fine tunning in presciption when typed ahead an ARV Drug to enable automatically the ARV checkBox

### CODE REVIEW

File: app/common/prescription/controllers/patientSimplifiedPrescriptionController.js [hunk 1]
```diff
@@ -225,6 +230,17 @@
         });
     }
 
+    function isArvDrug(drug) {
```
Comment (ynurmahomed) [inline app/common/prescription/controllers/patientSimplifiedPrescriptionController.js]: This function should be in [drugService](https://github.com/esaude/esaude-emr-poc/blob/master/app/common/domain/services/drugService.js). We should follow the [styleguide](https://github.com/johnpapa/angular-styleguide/tree/master/a1#data-services).

File: app/common/prescription/views/prescription-arv.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h3 class="panel-title">{{'CLINIC_PRESCRIPTION_CURRENT_REGIMEN'| translate}}</h3
 
             <!-- ARV Plan elements -->
             <div class="form-group form-group-lg"
-                         ng-class="{'has-error': prescriptionForm.artPlan.$invalid && vm.showMessages}">
+                         ng-class="{'has-error': prescriptionForm.artPlan.$invalid && vm.showMessages}" ng-init="vm.initArvPlans()">
```
Comment (ynurmahomed) [inline app/common/prescription/views/prescription-arv.html]: I would prefer that ```initArvPlans``` is called inside the controllers ```activate``` function. [syleguide](https://github.com/johnpapa/angular-styleguide/tree/master/a1#style-y080).

File: app/common/prescription/controllers/patientSimplifiedPrescriptionController.js [hunk 3]
```diff
@@ -126,16 +127,20 @@
...
-      var arvRepr = drugMapping.arvDrugs[drug.uuid];
 
-      if (arvRepr) {
-        vm.prescriptionItem.isArv = true;
-      } else {
-        vm.prescriptionItem.isArv = false;
-        vm.prescriptionItem.interruptedReason = {};
-        vm.prescriptionItem.isPlanInterrupted = false;
-        vm.prescriptionItem.arvPlan = {};
+      if(vm.prescriptionItem.drugOrder.drug || !vm.prescriptionItem.drugOrder){
```
Comment (ynurmahomed) [inline app/common/prescription/controllers/patientSimplifiedPrescriptionController.js]: This should be easy to unit test.

File: app/common/prescription/controllers/patientSimplifiedPrescriptionController.js (hunk 1 above)
Comment (Guimino) [inline app/common/prescription/controllers/patientSimplifiedPrescriptionController.js]: I followed your recomendations.

File: app/common/prescription/views/prescription-arv.html (hunk 2 above)
Comment (Guimino) [inline app/common/prescription/views/prescription-arv.html]: Great!.  I think later we can find time and do this refactory. For now please merge my PR

**Comment by ynurmahomed** [comment]
[1.0.1_28 Feature: Cancel Dispensation](https://trello.com/c/9Ab1hdU3/85-10128-feature-cancel-dispensation)

**Pull request merged**